.github/workflows/integrate.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
     - name: Install dependencies
       run: |
         set -eux
-        sudo pip3 install charmcraft
+        sudo pip3 install charmcraft==1.0.0
```

Comment:
This really should be switched to the snap, but this is reasonable to keep the workflows passing and the scope of this PR down.